pyeudiw/oauth2/dpop.py

```diff
@@ -81,12 +80,24 @@ def __init__(
             if self.dpop_header_prefix in http_header_authz
             else http_header_authz
         )
+        # If the jwt is invalid, this will raise an exception
+        try:
+            unpad_jwt_header(http_header_dpop)
+        except UnicodeDecodeError:
```

Comment:
please add a generic Exception after this, in the case that an unhandled exception may occour

do you agree?

Reply:
Resolved in 560e274